pkg/rook/status.go

```diff
@@ -36,11 +36,12 @@ func PrintCustomResourceStatus(clusterNamespace string, arg []string) {
 	if len(arg) == 1 && arg[0] == "all" {
 		command := fmt.Sprintf(getCrdList, clusterNamespace)
 		allCRs := strings.Split(exec.ExecuteBashCommand(command), "\n")
-		logging.Info(allCRs[0])
+		allCRs = allCRs[:len(allCRs)-1]
 		for _, cr := range allCRs {
 			logging.Info(cr)
 			command := fmt.Sprintf(scriptPrintSpecificCRStatus, clusterNamespace, cr)
-			logging.Info(exec.ExecuteBashCommand(command))
```

Comment:
The `logging.Info()` is printing "Info:" with no other info for other commands besides the example given in the issue. Instead of changing the implementation here, try just change inside the `logging.Info()` method to only print a blank line if there is no message.